entgql/internal/todoplugin/ent/schema/category.go

```diff
@@ -0,0 +1,88 @@
+// Copyright 2019-present Facebook
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package schema
+
+import (
+	"time"
+
+	"entgo.io/contrib/entgql"
+	"entgo.io/ent"
+	"entgo.io/ent/dialect"
+	"entgo.io/ent/schema"
+	"entgo.io/ent/schema/edge"
+	"entgo.io/ent/schema/field"
+	"github.com/google/uuid"
+
+	"entgo.io/contrib/entgql/internal/todoplugin/ent/schema/schematype"
+)
+
+// Category holds the schema definition for the Category entity.
+type Category struct {
```

Comment:
Can't you import this schema from the `todo` package?

Reply:
I would like to add some special annotations to this scheme in the future like: `RelayConnection`, `Directives`, `Interfaces`,... So, I don't want to touch on the current example.

On the other hand, some JSON data types are not yet supported at GQL generation. For example: `Strings()`, `Ints()`, and `Floats()`.

Reply:
> I would like to add some special annotations to this scheme in the future like: RelayConnection, Directives, Interfaces,... So, I don't want to touch on the current example.

You can import another schema and use it as a Mixin. See examples in internal.

> On the other hand, some JSON data types are not yet supported at GQL generation. For example: Strings(), Ints(), and Floats().

Got it, so let's skip them for now instead of failing. It's preferred instead of copy-pasting the examples.